ci/jenkins/test.sh

```diff
@@ -375,7 +375,7 @@ function revert_snapshot_windows {
 function  build_and_deliver_antrea_windows_and_linux_containerd_images {
     echo "====== Cleanup Antrea Installation Before Delivering Antrea Windows and Antrea Linux containerd Images ====="
     clean_antrea
-    kubectl delete -f ${WORKDIR}/antrea-windows-containerd-with-ovs.yml --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/antrea-${WINDOWS_YAML_SUFFIX}.yml --ignore-not-found=true || true
     kubectl delete -f ${WORKDIR}/kube-proxy-windows-containerd.yml --ignore-not-found=true || true
```

Comment:
We don't need kube-proxy in windows CI any more because both userspace and pristine tests have enabled proxy-all in agent. Please remove all related parts, including the `apply` command in `wait_for_antrea_windows_pods_ready`.

Reply:
Also remove kube-proxy related images in image pulling code.